.ci/opensearch/Dockerfile

```diff
@@ -1,10 +1,17 @@
-ARG OPENSEARCH_VERSION
+ARG OPENSEARCH_VERSION=latest
 FROM opensearchproject/opensearch:$OPENSEARCH_VERSION
 
+# OPENSEARCH_VERSION needs to be redefined as any arg before FROM is outside build scope.
+ARG OPENSEARCH_VERSION=latest
```

Comment:
line 1 & 5 seems same

Reply:
So, this is required here. It seems that any `ARG` that is defined before `FROM` won't be available after `FROM`.  [Docker Docs ](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact) say the same. Since, we need the `OPENSEARCH_VERSION` for updating config file

Reply:
Awesome, could you also please add this doc link in the code too, so that it won't be confusing later?